.travis.yml

```diff
@@ -14,17 +14,18 @@ branches:  # exclude tags
 env:
   global:
     # compiler command for Travis
-    - CXX=g++-4.8
+    # disabled since not using native modules
+    # - CXX=g++-4.8
     # build number used by blt pipeline
     - CI_BUILD_NUMBER=$TRAVIS_BUILD_NUMBER
     - secure : "PwdqejbAT+2GcRDbZEgFayjV24iQvMqNDFhrVS4mc6LF4x3GxDflkvM0f/hzP+Gz5Nk3Aj9EwAh3ULjrw4jV7cIjepJRiBNZ2CVne8k3LyYBxYydCTekLaArjBczAUpTI8tzAFgpYJdLGi7aZ4e+5dNIC8CFNMIrvs4VFTWK6iIRi/QcD0axCrPVL8imCHD1lYv+1P9rkRTLXPGELVUdxR3tk6iEASgcWiD9GgzG38tw53fZNAX8udTtY8sPhqNsTd3TajMBfgClsW72qFpX1hi0bj97xyp/O0sNslfS3+OuAtc4TGXVHD9YXrPcANXrM0uCCl/o1b7JW1cdp2kyENnSXCK1DK5JHCByFiGQ7dpMRdDptL2FXkB2m1Hmm50APjuKq2TCqr/ew1r4yCiAwrcTZtyz21SXkRGbCqVvR3A3SagdA7pJYpNbkNG9IJznbxZI6dY3TWcKWfSnITguErnZ0Y7NDfqwBCI5KUKrsJtBLmFeJPx88UVEaz2Gi6F53I64twyMQZv7Nbmp7jSVYWVPe9Q/6bfPBe8OPdKL1vJ+YdnGtHvbhmol6UKZ1hTaDCNuF1JdP1PBdMhntRzly2W1X4PzvYPsGEfvYrkOBPTdCkxl/5HKJ1X9IQdQIODN5tTnUzngZFxMnaNrI0W03jRu3xbpTyxL2iWwWR71g24="
 
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - g++-4.8
+# addons:
+#   apt:
+#    sources:
+#      - ubuntu-toolchain-r-test
+#    packages:
+#      - g++-4.8
```

Comment:
Travis has our back - this is part of the default output for Node builds that don't specify a compiler:

![screen shot 2017-04-13 at 11 53 24 am](https://cloud.githubusercontent.com/assets/1885153/24984158/e4fe16ea-203f-11e7-83a2-2cbf9c67c80d.png)

That link [has all the info anyone would need](Starting with io.js 3 and Node.js 4, building native extensions requires C++11-compatible compiler, which seems unavailable on this VM. Please read https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements.)